### PR TITLE
Feature/delete unused flags

### PR DESCRIPTION
**Pull request by Trjegul84**

### CODE REVIEW

**Comment by cazapatamar** [comment]
We shouldn't delete concentration flag and enable this section as it is still pointing to mock data.

**Comment by erikasv** [comment]
> We shouldn't delete concentration flag and enable this section as it is still pointing to mock data.

Agree, but please wait, I'm solving conflicts

**Comment by erikasv** [comment]
Done, conflicts solved. Please restore the concentration flag. Sorry for putting it in the list in the first place :sweat_smile:

File: src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx [hunk 1]
```diff
@@ -335,7 +335,7 @@ class SpeciesRecordsGaps extends React.Component {
                   {getLabelConcentration(key, areaId)}
                 </LineLegend>
 
-              ))}
+                ))}
```
[the remark this answers is not in the document]
Comment (erikasv) [reply src/pages/search/drawer/species/richness/SpeciesRecordsGaps.jsx]: done

**Review by cazapatamar — APPROVED** [review]

**Pull request merged**